Libraries/Network/RCTNetworking.windows.js

```diff
@@ -58,7 +56,8 @@ class RCTNetworking extends NativeEventEmitter {
     responseType: 'text' | 'base64',
     incrementalUpdates: boolean,
     timeout: number,
-    callback: (requestId: number) => any
+    callback: (requestId: number) => any,
+    withCredentials: boolean
   ) {
```

Comment:
Remove for now, but we should add this.

Reply:
RNTester working once removed.

---
In reply to: [169649317](https://github.com/Microsoft/react-native-windows/pull/1659#discussion_r169649317) [](ancestors = 169649317)